content/docs/19-kubernetes-knowlege-hub/00-how-to/06-deploy-stateless-frontend-app.md

```diff
@@ -0,0 +1,208 @@
+
+COPY . .
+
+EXPOSE 3000
+
+CMD ["npm", "start"]
+```
+
+Also, create a **.dockerignore** file and add the following lines to it.
+
+```bash
+/node_modules
+/.pnp
+.pnp.js
+/coverage
+```
+
+## Build a Docker Image of the Application.
+
+This step packages the application into a portable image. To build the app’s image, run the Docker `build` command as shown:
+
+```bash
+docker build -t date-suggestions .
+```
+
+## Create a Kubernetes Deployment.
+
+Before continuing with this step, ensure that you have access to a Kubernetes cluster, as explained in the [prerequisites](https://www.notion.so/How-To-Deploy-A-Stateless-Frontend-App-with-Kubernetes-b885ae2307e94ef191a1b713fe29c81f). 
+
+In the application's root directory, create a Kubernetes Deployment file using the `kubectl` command below:
+
+```bash
+kubectl create deploy date-suggestions --image=date-suggestions --replicas=2 --port=3000 --dry-run=client -o yaml
+```
+
+The output of running the command above is a YAML representation of the deployment, likely similar to the lines below:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: date-suggestions
+spec:
+  selector:
+    matchLabels:
+      app: date-suggestions
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: date-suggestions
+    spec:
+      containers:
+        - name: date-suggestions
+          image: date-suggestions
+          ports:
+            - containerPort: 3000
+```
+
+ 
+modify the command to create the deployment, to add a command to pipe the output into a `deployment.yaml` file.
+
+```bash
+kubectl create deploy date-suggestions --image=date-suggestions --replicas=2 --port=3000 --dry-run=client -o yaml > deployment.yaml
+
+```
+
+Alternatively, you can use the `touch` command to create the `deployment.yaml` file, and then copy the YAML output from the command to create a deployment to it.
+
+```bash
+touch deployment.yaml
+```
+
+## Create a Kubernetes Service.
+
+Create and populate a Kubernetes Service file in the app's root directory. As the default, your application will only be accessible within the cluster. You'll need to create a Kubernetes service resource to expose it to the outside world. A service resource creates an abstraction over a set of pods that provides discovery and routing between them.
+
+To create a service, use the `kubectl expose` command as shown:
+
+```bash
+kubectl expose deployment date-suggestions --type=LoadBalancer --port=80 --target-port=3000 --name=date-suggestion-service --dry-run=client -o yaml
+```
+
+The output of running the command will be similar to the YAML below:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: date-suggestions-service
+spec:
+  type: LoadBalancer
+  selector:
+    app: date-suggestions
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 3000
+```
+
+If everything looks good, modify the command to expose the deployment, to add a command to add the output to a file `service.yaml`.
```

Comment:
```suggestion
If everything looks good, modify the command to redirect the output YAML to the file **service.yaml**.
```
Similar to earlier, I think you want the redirect operator. I also shortened the sentence to make it easier to read.